decoimpact/data/parsers/parser_axis_filter_rule.py

```diff
@@ -47,16 +46,23 @@ def parse_dict(self, dictionary: Dict[str, Any], logger: ILogger) -> IRuleData:
             )
             raise ValueError(message)
 
-        layer_number = get_dict_element("layer_number", dictionary)
-        if not isinstance(layer_number, int):
+        element_index = get_dict_element("layer_number", dictionary)
+        if not isinstance(element_index, int):
             message = (
                 "Layer number should be an integer, "
-                f"received a {type(layer_number)}: {layer_number}"
+                f"received a {type(element_index)}: {element_index}"
             )
             raise ValueError(message)
         output_variable_name = get_dict_element("output_variable", dictionary)
 
```

Comment:
HHere we have items on separate lines and before we had a fairly lengthy line with all arguments compressed on it. Seems inconstent to me.